Added initial cloud files (#7338)
influxdata/docs-v2: content/shared/influxdb3-get-started/setup-cloud.md

<!-- COMMENT TO ALLOW STARTING WITH SHORTCODE -->

{{% product-name %}} is the fully managed, cloud-hosted version of
[InfluxDB 3 Enterprise](/influxdb3/enterprise/).
InfluxData provisions and operates the InfluxDB 3 servers for you, so you don't
install, run, or scale servers yourself.
To get started, get your instance host from InfluxData, configure the
`influxdb3` CLI, log in, and then create a database.

- [Get your instance](#get-your-instance)
- [Configure the influxdb3 CLI](#configure-the-influxdb3-cli)
  - [Connect to your instance](#connect-to-your-instance)
  - [Log in to your instance](#log-in-to-your-instance)
- [Create a database](#create-a-database)
- [Create a token for applications](#create-a-token-for-applications)

## Get your instance

During early access, InfluxData provisions {{% product-name %}} instances for

> you--there's no self-service signup.

Reviewer: Use two sentences, and "sign up" as a verb.

```suggestion
During early access, InfluxData provisions {{% product-name %}} instances for
you. You can't sign up for the service yourself.
```

To request early access, see

> [InfluxDB 3 Cloud](https://www.influxdata.com/products/influxdb3-cloud/).

Reviewer: ```suggestion
To request early access, see the
[InfluxDB 3 Cloud product page](https://www.influxdata.com/products/influxdb3-cloud/).
```

After your instance is provisioned, InfluxData provides your instance
**host URL**--the endpoint you connect to.
You authenticate as a user with your InfluxData account in the following

> steps--there's no token to copy or store for interactive use.

Reviewer: Use separate sentences here.

```suggestion
After your instance is provisioned, InfluxData provides your instance
**host URL**, which is the endpoint you connect to.
You authenticate with your InfluxData account in the following steps.
You don't copy or store a token for interactive use.
```

## Configure the influxdb3 CLI

The [`influxdb3` CLI](/influxdb3/cloud/reference/cli/influxdb3/) lets you
administer your instance and write and query data from the command line.
Because {{% product-name %}} is fully managed, there's nothing to install or run
for {{% product-name %}} itself--you point the `influxdb3` CLI at your hosted

> instance and log in with your InfluxData account.

Reviewer: Use a separate sentence, and refer to "the service" to avoid repeating the product name.

```suggestion
Because {{% product-name %}} is fully managed, you don't install or run
anything for the service itself. You point the `influxdb3` CLI at your hosted
instance and log in with your InfluxData account.
```

### Connect to your instance

The `influxdb3` CLI connects to `http://127.0.0.1:8181` by default, so set the

> `INFLUXDB3_HOST_URL` environment variable to your instance host URL:

Reviewer: Lead with the action for this step.

```suggestion
Set the `INFLUXDB3_HOST_URL` environment variable for the `influxdb3` CLI to connect to your instance host URL:
```

{{< tabs-wrapper >}}
{{% tabs %}}
[macOS and Linux](#)
[PowerShell](#)
[CMD](#)
{{% /tabs %}}
{{% tab-content %}}

```sh
export INFLUXDB3_HOST_URL=https://{{< influxdb/host >}}
```

{{% /tab-content %}}
{{% tab-content %}}

```powershell
$env:INFLUXDB3_HOST_URL = "https://{{< influxdb/host >}}"
```

{{% /tab-content %}}
{{% tab-content %}}

```sh
set INFLUXDB3_HOST_URL=https://{{< influxdb/host >}} 
# Make sure to include a space character at the end of this command.
```

{{% /tab-content %}}
{{< /tabs-wrapper >}}

### Log in to your instance

<span id="set-your-token-for-authorization"></span>

Log in to your instance with the OAuth device-code flow.
{{% product-name %}} uses your InfluxData account for authentication, so you

> don't manage static tokens for interactive use.

Reviewer: State it directly: with OAuth login, you don't need a token for interactive CLI use.

```suggestion
{{% product-name %}} authenticates you with your InfluxData account, so you
don't need a token to use the CLI interactively.
```

Run the following command and follow the printed instructions:

<!--pytest.mark.skip-->

```sh
influxdb3 auth login --oauth
```

> The command prints a URL and a one-time user code.

Reviewer: ```suggestion
The output is a URL and a one-time user code.
```

Open the URL in a browser, enter the code, and approve the request.
After you approve, the CLI saves your credentials to
`~/.influxdb3/credentials.json` and uses them automatically for subsequent

> commands--you don't need to pass a token.

Reviewer: Join with "so," and use "later" instead of "subsequent."

```suggestion
After you approve, the CLI saves your credentials to
`~/.influxdb3/credentials.json` and uses them automatically for later
commands, so you don't need to pass a token.
```

> [!Note]
> Don't set the `INFLUXDB3_AUTH_TOKEN` environment variable for interactive use.
> If it's set, the `influxdb3` CLI uses that token instead of your logged-in
> session. Run `influxdb3 auth login --oauth` again when your session expires.

> Verify that you're authenticated by listing your databases:

Reviewer: ```suggestion
List your databases to verify that you're authenticated:
```

<!--pytest.mark.skip-->

```sh
influxdb3 show databases
```

A successful response confirms that your host URL and login are configured
correctly.

## Create a database

Use the

> [`influxdb3 create database` command](/influxdb3/cloud/reference/cli/influxdb3/)

Reviewer: TODO: update this URL when reference content is added

to create a database.
You can use an existing database or create a new one specifically for this
getting started tutorial.
_Examples in this tutorial assume a database named `get-started`._

Provide the following:

- The database name
- _Optional:_ a database

>   [retention period](/influxdb3/cloud/admin/databases/)

Reviewer: ```suggestion
  [retention period](/influxdb3/cloud/admin/databases/#retention-periods)
```

  as a duration value. If no retention period is specified, the default is
  infinite.

<!--pytest.mark.skip-->

```sh { placeholders="get-started|1y" }
influxdb3 create database \
  --retention-period 1y \
  get-started
```

Replace the following:

- {{% code-placeholder-key %}}`get-started`{{% /code-placeholder-key %}}:
  the name of the database to create
- {{% code-placeholder-key %}}`1y`{{% /code-placeholder-key %}}:
  the database retention period as a duration

> [!Note]
> A newly created database can take a few seconds to appear in
> `influxdb3 show databases`.

## Create a token for applications

Logging in authenticates you for interactive use with the `influxdb3` CLI.
For applications and automated clients, create a **database token** scoped to
only the databases and permissions it needs.
Creating tokens requires admin privileges--if your user doesn't have admin

> access, ask an administrator to create a token for you.

Reviewer: Use two sentences.

```suggestion
Creating tokens requires admin privileges. If your user doesn't have admin
access, ask an administrator to create a token for you.
```

Use the
[`influxdb3 create token` command](/influxdb3/cloud/reference/cli/influxdb3/)
with the `--permission` option to create a token with read and write access to
your database.

Provide the following:

- `--permission`: A resource permission string in the format
  `db:DATABASE_NAME:read,write`
- `--name`: A unique name for the token

<!--pytest.mark.skip-->

```sh { placeholders="get-started" }
influxdb3 create token \
  --permission "db:get-started:read,write" \
  --name "Read/write token for get-started database"

> ```
> 
> Replace {{% code-placeholder-key %}}`get-started`{{% /code-placeholder-key %}}
> with the name of the database to grant the token access to.
> 
> The command returns the token string.
> Store the token string in a safe place.
> **This is the only time the token string is available in plain text.**
> Applications can authenticate with the token by setting the
> `INFLUXDB3_AUTH_TOKEN` environment variable or passing the `--token` option.

Reviewer: Use the standard "Store your token securely" callout, matching `setup.md`.

```suggestion
The command returns the token string.

> [!Important]
> #### Store your token securely
>
> InfluxDB displays the token string only when you create it.
> Store your token securely—you cannot retrieve it from the database later.

Applications can authenticate with the token by setting the
`INFLUXDB3_AUTH_TOKEN` environment variable or passing the `--token` option.
```
